workers/tests/integrationtests/test_api_regions.py

```diff
@@ -11,16 +11,51 @@
 from ohsome_quality_analyst.api import app
 
 
-class TestApiIndicator(unittest.TestCase):
+class TestApi(unittest.TestCase):
     def setUp(self):
         self.client = TestClient(app)
 
     def test_get_available_regions(self):
-        url = "/regions"
+        url = "/list_regions"
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
         self.assertTrue(geojson.loads(response.content).is_valid)
 
+    def test_get_list_indicators(self):
+        url = "/list_indicators"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsInstance(geojson.loads(response.content), list)
```

Comment:
The response shouldn't be of type list as it is defined right now.

The same applies to the following tests.